docs/user-guide/time-and-costs/progress-tracking/README.md

```diff
@@ -2,93 +2,111 @@
 
-## Progress tracking in the work package hierarchy
+### Status-based progress reporting
 
-The progress of **work package with children** is calculated as the weighted average of all direct children, using the field **Work (earlier called Estimated time)** as the weight.
-When adding the **% Complete (earlier called Progress (%))** column to a work package hierarchy view, please also add the **Work**
-column as well so that you can track the calculation.
-The screenshot below shows an example hierarchy with aggregated **% Complete**.
+Administrators can also switch to [status-based progress reporting mode](../../../system-admin-guide/manage-work-packages/work-package-settings/) for their instance. 
 
-Please note the **Work** column to the right.
-Here, values next to the sum icon (for example 43 h next to the  Main task) indicate the sum of estimated time of the task itself and its children, while values without the sum icon (3 hours) refer to estimated time directly assigned to a work package.
+In this mode, each status is associated with a fixed %&nbsp;Complete value in the [admin settings for each one]((../../../system-admin-guide/manage-work-packages/work-package-status/)), which can be freely updated by changing the status of a work package. This allows teams to report progress simply by changing the status of their work packages over time.
 
-![Progress calculation in the WP hierarchy](progress-tracking-hierarchy-progress.png)
+> For example, "New" could be set to 0%, "In progress" to 30%, "In test" to 60% and "Implemented" to 100%. Then, as project members update the status, the %&nbsp;Complete values would reflect these changes automatically.
 
-Calculation examples:
- * Sub-Task 1 has 50% completion calculated as (16h * 60% + 8h * 30%) / 24h.
- * Sub-Task 2 has 25% completion calculated as (8h * 50% + 8h * 0%) / 16h.
- * Main Task has 40% completion calculated as (24h * 50% + 16h * 25%) / 40h.
+Unlike in work-based progress reporting mode, in status-based mode, Remaining work is an automatically calculated value that cannot be manually edited.
 
-## Status-based progress tracking
+>**Remaining work** is **Work** times **(100% - %&nbsp;Complete)**, expressed in hours. For example, if the %&nbsp;Complete for a selected status is 25% and Work is 20h, Remaining work is automatically set to 15h.
 
-As an alternative to the manual progress tracking mode above, you can configure your OpenProject system to associate work packages statuses with fixed values for % Complete .
-In the Administration, please navigate to Work packages -> Settings and change “Calculate the work package done ratio” to
-“Use the work package status”.
+In Status-based progress reporting mode, Work is not a required value. However, if Work is set, Remaining work is automatically calculated. To make this link clear and transparent, clicking on any of the three values to modify them will display the following pop-over:
```

Comment:
Actually, clicking on Remaining work or % Complete field will do nothing because they are read-only. Only clicking on Work will display the popover.

Reply:
Fixed manually.